.eslintrc

```diff
@@ -11,5 +11,8 @@
   },
   "plugins": [
     "react"
-  ]
-}
\ No newline at end of file
+  ],
+  "rules": {
+    "no-else-return": "off"
```

Comment:
id prefer to keep this rule for consistency with our other code bases for now - trying to keep all the eslint configs in sync 🙏

Reply:
Agreed though that we can turn this on in the near future